machine-learning/00_series-intro_outline.md

```diff
@@ -0,0 +1,125 @@
+# Machine Learning
+
+(!) = Definitely needs to be reworded or explained more
+
+Standardized CodeRabbit introduction
+
+Introduction
```

Comment:
I'm thinking that this section on its own would make a good ~1.5ish minute introduction